pyproject.toml

```diff
@@ -0,0 +1,125 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mozilla-django-oidc-db"
+version = "0.18.1"
+description = "A database-backed configuration for mozilla-django-oidc"
+authors = [
+    {name = "Maykin Media", email = "support@maykinmedia.nl"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+keywords = ["OIDC", "django", "database", "authentication"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
that's not possible with a `0.x` version, and we are planning more breaking changes so bumping to 1.0 is not possible yet :)